backend/infrastructure/gateways/PostgreSQLLieuLoader.test.ts

```diff
@@ -69,9 +69,10 @@ describe('lieu loader', () => {
       const longitude = 2.000000
       const page = 0
       const nombreDeLieuxAffichesParPage = 4
+      const rayonDeRechercheIllimité = 10000
```

Comment:
```suggestion
      const rayonDeRechercheIllimite = 10000
```
Pareil pour les autres.